src/utiltest.cpp

```diff
@@ -151,3 +155,52 @@ CWalletTx GetValidSpend(ZCJoinSplit& params,
     CWalletTx wtx {NULL, tx};
     return wtx;
 }
+
+// Sapling
+const Consensus::Params& ActivateSapling() {
+    SelectParams(CBaseChainParams::REGTEST);
+    UpdateNetworkUpgradeParameters(Consensus::UPGRADE_OVERWINTER, Consensus::NetworkUpgrade::ALWAYS_ACTIVE);
+    UpdateNetworkUpgradeParameters(Consensus::UPGRADE_SAPLING, Consensus::NetworkUpgrade::ALWAYS_ACTIVE);
+    return Params().GetConsensus();
+}
+
+void DeactivateSapling() {
+    UpdateNetworkUpgradeParameters(Consensus::UPGRADE_SAPLING, Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT);
+    UpdateNetworkUpgradeParameters(Consensus::UPGRADE_OVERWINTER, Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT);
+    // Consider: Should we be doing the following?
+    // SelectParams(CBaseChainParams::MAIN);
```

Comment:
I don't think so. If I understand correctly, the selected network is unspecified when entering a test, so every test should explicitly select the right one.

Reply:
I realize that I'm not sure what the effect will when we are doing this in the benchmarks, but they seem to work fine without it. I will push a commit to update the comment and remove `ActivateSapling()`/`DeactivateSapling()` from the benchmarks.

Reply:
I would suggest leaving `ActivateSapling()` in the benchmarks; it can't do any harm and some of the code we're benchmarking might depend on it (or might change to depend on it in future).